### ISSUE TITLE
Allow api version override in azmetrics client

### ISSUE THREAD

**Issue by PCaponetti** [Azure/azure-sdk-for-go#23819]
### Feature Request

Please enable override of the API version in azmetrics. It seems that outside of testing and validation (which I know may be significant), all that is required is to set pipeline options with the following: 
```
APIVersion: runtime.APIVersionOptions{
	Location: runtime.APIVersionLocationQueryParam,
	Name:     "api-version",
}
```

**Comment by gracewilcox** [comment]
Hi @PCaponetti! Thank you for your request! We can work on enabling API version override in a future release.

**Comment by gracewilcox** [comment]
Released today in [version 1.2.0](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/query/azmetrics@v1.2.0)!

**Comment by PCaponetti** [comment]
Thanks!
